classes/class-p4-search.php

```diff
@@ -617,15 +704,15 @@ protected function set_filters_context( &$context ) {
 			}
 
 			// Sort associative array with filters alphabetically.
-			if ( $context['categories'] ) {
+			if ( $context['categories'] ?? false ) {
```

Comment:
Probably doesn't occur in prod, but I set up a new language which didn't have categories and this was producing warnings.